controller/api/management/commands/load_db_state_to_etcd.py

```diff
@@ -0,0 +1,11 @@
+from django.core.management.base import BaseCommand
+from api.models import Key, App, Domain, Certificate, Config
```

Comment:
[PEP8](https://www.python.org/dev/peps/pep-0008/#imports) says there should be a blank line here, between third-party and local imports. (Although the `flake8` tool doesn't enforce this.)